internal/constants/constants.go

```diff
@@ -0,0 +1,15 @@
+package constants
```

Comment:
I feel like this isn't really descriptive for what it is. Could we move this under internal/guest still as well?

Reply:
I'm using some of the constants outside of `internal/guest`. I just wanted to make sure we're using the same values in both hcsshim and gcs code paths. do you have any suggestion on naming here, cuz it's hard? 😄

Reply:
hmm I see. Maybe we should also move out some of the other guest root path related values from internal/uvm/constants.go as well. Maybe we could call the package `guestpath`? Naming is hard haha

Reply:
sure, `guestpath` sgtm

Reply:
done!